test/log/test_console_adapter.rb

```diff
@@ -0,0 +1,110 @@
+require_relative '../helper'
+
+require 'fluent/log'
+require 'fluent/log/console_adapter'
+
+class ConsoleAdapterTest < Test::Unit::TestCase
+  def setup
+    @timestamp = Time.parse("2023-01-01 15:32:41 +0000")
```

Comment:
nitpick: All of unit tests is quite reasonable for me. However, using UTC TimeZone is a sort of overlooking of time zone bug. Shouldn't we use (-04:00) EST or (+09:00) JST? Leaving as-is is also OK for me.

Reply:
Thanks for you comment!
I though similar thing when writing a test but I think it should be covered by tests for `Fluent::Log`.
So I won't fix this, leave as-is.